apps/shopify/src/productPagination.js

```diff
@@ -8,12 +8,16 @@ const makePagination = async (sdk) => {
     fetchProducts: async function (search, PER_PAGE) {
       const query = { query: `variants:['sku:${search}'] OR title:${search}` };
 
-      return await this.shopifyClient.product.fetchQuery({
+      const response = await this.shopifyClient.product.fetchQuery({
         first: PER_PAGE,
         sortBy: 'TITLE',
         reverse: true,
         ...(search.length && query),
       });
+      // converting the 'gid://shopify/ProductVariant/41' to base64 format as API format changed from 2022-04 version
+
+      const products = response.map((res) => { return { ...res, id: Buffer.from(res.id).toString('base64') } })
+      return products
```

Comment:
Since we seem to be doing the same over and over, should we try extract a util which encapsulates the need of base64 conversion?